test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -91,6 +92,9 @@ private void assertCommandBehavior(String inputCommand,
         assertEquals(expectedAddressBook, addressBook);
         assertEquals(lastShownList, logic.getLastShownList());
         assertEquals(addressBook, saveFile.load());
+        if (parsedCommand.isMutating()) {
```

Comment:
You should test the else branch where the command is not mutating. Some thought process is required to come up with the way of testing.